sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -1,70 +1,5 @@
-# Azure Key Vault Package client library for C++
```

Comment:
The boiler plate references for contributing should stay in this file.

Reply:
Yes, I just leave it all empty for now since it all needs to be written carefully again.
But yes, I will revert the contributing and any other part that is just the same as all other packages